tests/actions/novels.test.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios'
```

Comment:
This test doesn't run because you told mocha to look for *.test.js which doesn't work for this .jsx file. You need to change that to *.test.js*